media/css/m24/feature.scss

```diff
@@ -0,0 +1,93 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+
+// feature
+.m24-c-feature {
+    @include container;
+    padding-bottom: $spacer-xl;
+}
+
+.m24-c-feature-media {
+    padding-top: $container-padding;
+    padding-bottom: $spacer-xl;
+}
+
+.m24-c-feature-title {
+    margin-bottom: $spacer-xl;
+    font-size: $alias-text-title-h2;
+    line-height: 1;
+    text-wrap: balance;
+    text-wrap-style: balance;
+}
+
+.m24-c-feature-cta {
+    text-align: end;
+}
+
+@media #{$mq-xl} {
+    .m24-c-feature-title {
+        max-width: grid(9);
+    }
+}
+
```

Comment:
I wasn't sure if I should move the column and picto code to their own .scss file or not. It's not being used anywhere else _YET_. And it's code that hangs off Protocol classes. Maybe it can stay here until the manifesto page is created?